sdk/avalon_sdk/work_order/work_order_request_validator.py

```diff
@@ -111,7 +113,8 @@ def validate_parameters(self, params):
                 not is_valid_hex_str(params["encryptedRequestHash"]):
             return False, "Invalid data format for encrypted request hash"
         if params["requesterNonce"] is None or \
-                not is_valid_hex_str(params["requesterNonce"]):
+                not is_valid_hex_str(params["requesterNonce"]) or \
+                len(params["requesterNonce"]) != 32:
```

Comment:
TC spec doesn't specify any length for requester nonce. Why do we want to check the length ?

Reply:
Added length check for params using constant in proxy model.